packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -194,15 +248,34 @@ class Doctor {
     return doctorResult;
   }
 
+  ValidationResult _mergeValidationResults(List<ValidationResult> results) {
```

Comment:
Hm.. This would break in weird ways if we ever have an additional validation result. Perhaps handle each result explicitly and throw if you see an unexpected one?

Reply:
Done.